config/nim.cfg

```diff
@@ -44,12 +44,12 @@ path="$lib/core"
 path="$lib/pure"
 
 @if nimbabel:
-  nimblepath="$home/.nimble/pkgs/"
   @if not windows:
     nimblepath="/opt/nimble/pkgs/"
   @else:
     # TODO:
   @end
+  nimblepath="$home/.nimble/pkgs/"
```

Comment:
Why?

Reply:
Should go in order - global to user to local deps. This originally put global deps after user deps.

Reply:
The user _should_ be able to override global deps.

Reply:
Then my change is correct cause nimble paths get added to the beginning of lazyPaths. In the original order, user packages got added followed by global ones which got inserted to the front.